week3/final/robot-text/robot.js

```diff
@@ -62,11 +69,15 @@
 
     const cellContents = board[y][x];
 
-    if (cellContents === '.' || cellContents === 'F') {
+    if (cellContents === '.' || cellContents === 'F' || cellContents==='A') {
       board[robot.y][robot.x] = trailIndicators[robot.dir];
       robot.x = x;
       robot.y = y;
       board[y][x] = 'R';
+      if (cellContents === 'A') {
+        appleEaten = true;
+        applesEaten ++;
```

Comment:
It is customary to append the `++` (or `--`) directly to the variable to which it applies (without an intervening space).